### ISSUE TITLE
Install not working on linux

### ISSUE THREAD

**Issue by raq929** [ga-wdi-boston/mongodb-study#13]
From Louis (cohort 016) Brew install mongodb did not work. Steps listed on MongoDB website also did not work.
https://docs.mongodb.com/v3.2/tutorial/install-mongodb-on-ubuntu/

**Comment by gaand** [comment]
@raq929 I don't believe this applies.  Please close if that is correct.  Or ask the developer to comment then close.

**Comment by raq929** [comment]
Will do.
